deploy/buildtools/bitbuilder.py

```diff
@@ -0,0 +1,320 @@
+import abc
+import yaml
+import json
+import time
+import random
+import string
+import logging
+import os
+from fabric.api import prefix, local, run, env, lcd, parallel # type: ignore
+from fabric.contrib.console import confirm # type: ignore
+from fabric.contrib.project import rsync_project # type: ignore
+
+from awstools.afitools import *
+from awstools.awstools import send_firesim_notification
+from util.streamlogger import StreamLogger, InfoStreamLogger
+
+from typing import Optional, TYPE_CHECKING
+# TODO: Solved by "from __future__ import annotations" (see https://stackoverflow.com/questions/33837918/type-hints-solve-circular-dependency)
+if TYPE_CHECKING:
+    from buildtools.buildconfig import BuildConfig
+else:
+    BuildConfig = object
+
+rootLogger = logging.getLogger()
+
+def get_deploy_dir() -> str:
```

Comment:
Can we avoid using Fabric to determine this path?

Reply:
This can be ignored for now.

Reply:
I think it would be better to do something with `__file__`like `return Path(__file__).parent.parent` and perhaps assert that what you looked up is a directory named `deploy` but I agree that it can wait.